### PR TITLE
Add docstrings for the docs API.

### PR DESCRIPTION
**Pull request by tdincer**

### CODE REVIEW

File: workflow_calcium_imaging/ingest.py [hunk 1]
```diff
@@ -17,8 +17,12 @@
 def ingest_subjects(
     subject_csv_path="./user_data/subjects.csv", skip_duplicates=True, verbose=True
```
Comment (CBroz1) [inline workflow_calcium_imaging/ingest.py:18]: ideally, this PR would include typing suggestions for each argument
```suggestion
    subject_csv_path:str="./user_data/subjects.csv", skip_duplicates:bool=True, verbose:bool=True
```

File: workflow_calcium_imaging/ingest.py [hunk 2]
```diff
@@ -137,10 +150,20 @@ def ingest_events(
     verbose=True,
 ):
     """
-    Ingest each level of experiment heirarchy for element-trial:
-        recording, block (i.e., phases of trials), trials (repeated units),
-        events (optionally 0-duration occurances within trial).
-    This ingestion function is duplicated across wf-array-ephys and wf-calcium-imaging
+    Ingest each level of experiment heirarchy for element-trial: recording, block (i.e.,
```
Comment (CBroz1) [inline workflow_calcium_imaging/ingest.py:153]:
```suggestion
    Ingest session, block, trial, and event data.

    Ingest each level of experiment hierarchy for element-trial: recording, block (i.e.,
```

File: workflow_calcium_imaging/ingest.py [hunk 3]
```diff
@@ -182,7 +205,14 @@ def ingest_events(
 def ingest_alignment(
     alignment_csv_path="./user_data/alignments.csv", skip_duplicates=True, verbose=True
 ):
-    """This is duplicated across wf-array-ephys and wf-calcium-imaging"""
+    """This is duplicated across wf-array-ephys and wf-calcium-imaging.
```
Comment (CBroz1) [inline workflow_calcium_imaging/ingest.py:208]:
```suggestion
    """Ingest event alignment information
    
    This is duplicated across wf-array-ephys and wf-calcium-imaging.
```

File: workflow_calcium_imaging/paths.py [hunk 4]
```diff
@@ -3,18 +3,30 @@
 
 
 def get_imaging_root_data_dir():
```
Comment (CBroz1) [inline workflow_calcium_imaging/paths.py:5]: Can this type suggest pathlib.Path?

```suggestion
def get_imaging_root_data_dir() -> SOMETHING:
```

File: workflow_calcium_imaging/paths.py [hunk 5]
```diff
@@ -65,13 +98,23 @@ def get_nd2_files(scan_key):
 
 
 def get_prairieview_files(scan_key):
+    """Retrieve the list of Prairieview files associated with a given Scan.
```
Comment (CBroz1) [inline workflow_calcium_imaging/paths.py:101]:
```suggestion
    """Retrieve the list of PrairieView files associated with a given Scan.
```

**Review by CBroz1 — CHANGES_REQUESTED** [review]

File: workflow_calcium_imaging/paths.py (hunk 4 above)
Comment (tdincer) [reply workflow_calcium_imaging/paths.py:5]: It returns either pathlib.Path or None.

**Review by CBroz1 — APPROVED** [review]
Looks good. Thanks @tdincer. Please confirm this is ready to merge

**Review by CBroz1 — APPROVED** [review]

**Pull request merged**